scripts/release/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+source $(dirname $0)/common.sh
+
+VERSION_NUMBER=$1
```

Comment:
Astuce: utiliser `${1:?Please specify release to deploy}` pour que le script s'arrête avec un message d'erreur en cas de paramètre manquant.